CHANGELOG.rst

```diff
@@ -13,6 +13,10 @@ Changelog
 ------
 - Fixed regression where ``GROUP BY`` class is missing for an aggregate with a specified order.
 
+0.16.8
```

Comment:
Minor issue: There is already a 0.16.8 header, could you merge the two sections?

Reply:
OK